content/en/docs/components/model-registry/installation.md

```diff
@@ -18,28 +18,39 @@ These are the minimal requirements to install Model Registry:
 
 ## Installing Model Registry
 
-You can skip this step if you have already installed Kubeflow >=1.9. Your Kubeflow
-deployment includes Model Registry ([see tracker issue](https://github.com/kubeflow/manifests/issues/2631)).
+Kubeflow Model registry may be installed as part of a Kubeflow Platform, or as a standalone component.
+The best option for you will depend on your specific requirements.
 
-To install Model Registry as part of Kubeflow, follow the
-[Kubeflow installation guide](/docs/started/installing-kubeflow/).
+### Installing on Kubeflow Platform
+
+Kubeflow Model Registry is available as an opt-in alpha component in Kubeflow Platform 1.9+, see [Installing Kubeflow](/docs/started/installing-kubeflow/) to learn more about deploying the Kubeflow Platform.
+
+If you have deployed the Kubeflow manifests, you may follow [these instructions](https://github.com/kubeflow/manifests/tree/master/apps/model-registry/upstream#readme) to deploy Model Registry; please raise any feedback on [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry/issues/new/choose).
```

Comment:
Usually, it's best to not link to the "new issue" page, so it encourages people to actually search existing issues first.

We should link to `https://github.com/kubeflow/model-registry/issues/` instead.

Reply:
done with 593e2335940bb6eacc5bbcffec34df12c2595730 thanks